CHANGELOG.md

```diff
@@ -1,6 +1,57 @@
 Change log
 ==========
 
+1.9.0 (2016-10-20)
+-----------------
+
+**Breaking changes**
+
+- For users of Docker Toolbox / Machine on Windows, the path conversion
+  for volumes is no longer done by default. To enable it, please set the
+  `COMPOSE_CONVERT_WINDOWS_PATHS` variable.
```

Comment:
We could be more explicit here. How about:

When using Compose with Docker Toolbox/Machine on Windows, volume paths are no longer converted from `C:\Users` to `/c/Users`-style by default. To re-enable this conversion so that your volumes keep working, set the environment variable `COMPOSE_CONVERT_WINDOWS_PATHS=1`. Users of Docker for Windows are not affected and do not need to set the variable.